internal/generate/collector/collect.go

```diff
@@ -47,6 +48,17 @@ type Manifests struct {
 	Others                           []unstructured.Unstructured
 }
 
+var (
+	roleGk                 = schema.GroupKind{Group: "rbac.authorization.k8s.io", Kind: "Role"}
+	clusterRoleGk          = schema.GroupKind{Group: "rbac.authorization.k8s.io", Kind: "ClusterRole"}
+	deploymentGk           = schema.GroupKind{Group: "apps", Kind: "Deployment"}
+	crdGk                  = schema.GroupKind{Group: "apiextensions.k8s.io", Kind: "CustomResourceDefinition"}
+	validatingWebhookCfgGk = schema.GroupKind{Group: "admissionregistration.k8s.io",
+		Kind: "ValidatingWebhookConfiguration"}
+	mutatingWebhookCfgGk = schema.GroupKind{Group: "admissionregistration.k8s.io",
+		Kind: "MutatingWebhookConfiguration"}
+)
+
```

Comment:
IMO would be nice to add a test to cover the scenario that you are trying to solve here. WDYT? Could you please add this one? Shows that it would be in : https://github.com/operator-framework/operator-sdk/blob/87e1fc4ee45ae8ff0ea6d02e3a6c9f47f41dbc0a/internal/generate/clusterserviceversion/clusterserviceversion_test.go#L332

Reply:
> IMO would be nice to add a test to cover the scenario that you are trying to solve here. WDYT?

Yes, I agree.

> Could you please add this one?

Done, Added a test case.